README.md

```diff
@@ -1,70 +1,71 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Space Travelers' Hub
 
-## Available Scripts
+> In this project, we have worked with the real live data from the SpaceX API. Our task was to build a web application for a company that provides commercial and scientific space travel services. The application allow users to book rockets, dragons and join selected space missions.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- React
+- Redux
+- React Boostrap
+- Deployed with netlify
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link](https://space-travelers-hub-1088.netlify.app/)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## GETTING STARTED
 
-### `npm run build`
+Clone this repo : git@github.com:CindyDorantes/Space-Travelers-Hub.git
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+After cloning this project, run the following commands:
```

Comment:
- [ ] Kindly include instructions in your ReadMe on how to run your test, as informed in the previous review.

Reply:
I'm really sorry and so embarrassed. Those changes were implemented and committed but I didn't realize I didn't push the code.